### PR TITLE
Updates to support in Microsoft Edge

### PR DESCRIPTION
**Pull request by tikhsuP**
- Added excluded URLs for edge.
- Updated README.md.

### CODE REVIEW

File: webextension/scripts/utils.js [hunk 1]
```diff
@@ -3,7 +3,7 @@
 let isArray = (a) => (!!a) && (a.constructor === Array)
 let isObject = (a) => (!!a) && (a.constructor === Object)
 
-let isFirefox = (navigator.userAgent.indexOf('Firefox') !== -1)
+let isFirefox = (navigator.userAgent.indexOf('Firefox') > -1)
```
Comment (anishsarangi) [inline webextension/scripts/utils.js:6]: Why did you change this ?

File: webextension/scripts/utils.js [hunk 2]
```diff
@@ -133,7 +133,8 @@ const excluded_urls = [
   'moz-extension:',
   '192.168.',
   '10.',
-  'file:'
+  'file:',
+  'edge:'
```
Comment (anishsarangi) [inline webextension/scripts/utils.js:137]: You also can add a new case in `isValidUrl` and `isNotExcludedUrl` in utils.spec.js

File: webextension/scripts/utils.js (hunk 1 above)
Comment (tikhsuP) [reply webextension/scripts/utils.js:6]: I was checking out MDN web docs documents and I found this: https://developer.mozilla.org/en-US/docs/Web/API/Window/navigator
They used ` > -1` so, I made the change.

File: webextension/scripts/utils.js (hunk 2 above)
Comment (tikhsuP) [reply webextension/scripts/utils.js:137]: Okay, will do it. Thanks :-)

File: webextension/scripts/utils.js (hunk 1 above)
Comment (tikhsuP) [reply webextension/scripts/utils.js:6]: Would you like me to revert it? :-)

Comment (anishsarangi) [reply webextension/scripts/utils.js:6]: No. You may create another variable and check for edge browser. Please ask Owenl to create new host name for edge browser. !
@cgorringe may need to check this once ! Thank you !

Comment (tikhsuP) [reply webextension/scripts/utils.js:6]: Yes, we need `edge-api`, and I will add the check for that. Thank you!

Comment (cgorringe) [reply webextension/scripts/utils.js:6]: We may want to rethink how we use isFirefox etc... as we'll have multiple browsers to support, including Edge and Safari.

Using `!== -1` vs `> -1` doesn't really make any difference. I'd leave it at `!== -1`
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/indexOf

File: webextension/scripts/utils.js (hunk 2 above)
Comment (tikhsuP) [reply webextension/scripts/utils.js]: I added the tests. Thank you!

File: webextension/scripts/utils.js (hunk 1 above)
Comment (tikhsuP) [reply webextension/scripts/utils.js]: I reverted this change. Thank you!
